### DISCUSSION TITLE
Curious question about using SD 1.5 models

### DISCUSSION THREAD

**Discussion by tomp199** [lllyasviel/Fooocus#3608] [Q&A] [answered]
I understand that it is possible to use SD 1.5 checkpoints and loras, but you also have to use a SDXL checkpoint at the same time. My question is, if your image is based on the SD 1.5 checkpoint, how important is it as to what the SDXL checkpoint is? For example you want to make a real to life image a a whale breaching out of the water, And for the SD 1.5 checkpoint you use I believe ones " epic photogasm". Does it matter what the SDXL checkpoint is? or does it not matter just as long as there is a random SDXL checkpoint in place?

**Comment by IPv6** [comment, marked as answer]
checkpoint matter, different checkpoints all have own stylistic biasing in choosing things. And LORAs appling with some difference to them.
So you need to test different checkpoints by yourself.

With Fooocus limit yourself to SDXL checkpoints, SD15 as refiner is not that useful, imho

**Reply by tomp199** [reply]
Thank you for the input.
